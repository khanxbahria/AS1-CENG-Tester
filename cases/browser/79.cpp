#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Browser* brows = new Browser();
Window a;
Window b;
Window c;
Window d;
Tab A = Tab("o", "v","qqq");
Tab B = Tab("kk", "l","ww");
Tab C = Tab("yy", "s","cc");
Tab D = Tab("dd", "xx","hh");
Tab E = Tab("k", "tt","hhh");
Tab F = Tab("j", "uu","qq");
Tab G = Tab("w", "w","dd");
Tab H = Tab("ww", "e","p");
Tab I = Tab("e", "eee","vv");
Tab J = Tab("x", "kkk","ooo");
Tab K = Tab("xxx", "xx","k");
Tab L = Tab("w", "kkk","jj");
Tab M = Tab("kk", "a","oo");
Tab N = Tab("v", "cc","o");
Tab O = Tab("qq", "i","j");
Tab P = Tab("yyy", "xxx","uu");
Tab Q = Tab("lll", "hhh","m");
Tab R = Tab("qqq", "ll","k");
Tab S = Tab("q", "y","ss");
Tab T = Tab("z", "eee","yyy");
Tab U = Tab("y", "k","rr");
Tab V = Tab("uu", "ddd","n");
Tab W = Tab("ww", "p","bbb");
Tab X = Tab("iii", "gg","v");
Tab Y = Tab("rr", "aa","bbb");
Tab Z = Tab("www", "g","ooo");
a.changeActiveTabTo(1);
b.moveActiveTabTo(1);
c.changeActiveTabTo(1);
d.changeActiveTabTo(1);
a.moveActiveTabTo(3);
b.changeActiveTabTo(3);
c.changeActiveTabTo(3);
d.moveActiveTabTo(3);
a.moveActiveTabTo(1);
b.newTab(A);
c.changeActiveTabTo(1);
d.changeActiveTabTo(1);
a.newTab(B);
b.changeActiveTabTo(3);
c.moveActiveTabTo(3);
d.changeActiveTabTo(3);
a.newTab(C);
b.newTab(D);
c.newTab(E);
d.changeActiveTabTo(1);
a.moveActiveTabTo(0);
b.newTab(F);
c.changeActiveTabTo(0);
d.newTab(G);
a.moveActiveTabTo(2);
b.newTab(H);
c.moveActiveTabTo(2);
d.newTab(I);
a.newTab(J);
b.newTab(K);
c.moveActiveTabTo(2);
d.newTab(L);
a.moveActiveTabTo(2);
b.moveActiveTabTo(2);
c.changeActiveTabTo(2);
d.changeActiveTabTo(2);
a.moveActiveTabTo(5);
b.changeActiveTabTo(5);
c.newTab(M);
d.newTab(N);
a.newTab(O);
b.newTab(P);
c.newTab(Q);
d.newTab(R);
a.changeActiveTabTo(0);
b.moveActiveTabTo(0);
c.changeActiveTabTo(0);
d.newTab(S);
a.newTab(T);
b.newTab(U);
c.newTab(V);
d.changeActiveTabTo(5);
a.changeActiveTabTo(0);
b.changeActiveTabTo(0);
c.changeActiveTabTo(0);
d.newTab(W);
a.moveActiveTabTo(4);
b.newTab(X);
c.moveActiveTabTo(4);
d.changeActiveTabTo(4);
a.newTab(Y);
b.moveActiveTabTo(1);
c.moveActiveTabTo(1);
d.newTab(Z);
a.moveActiveTabTo(1);
b.moveActiveTabTo(1);
d.changeActiveTabTo(1);
b.changeActiveTabTo(0);
c.changeActiveTabTo(0);
d.moveActiveTabTo(0);
a.changeActiveTabTo(2);
c.changeActiveTabTo(2);
d.moveActiveTabTo(2);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.changeActiveTabTo(3);
b.changeActiveTabTo(3);
c.changeActiveTabTo(3);
d.changeActiveTabTo(3);
a.changeActiveTabTo(5);
c.moveActiveTabTo(5);
d.changeActiveTabTo(5);
a.moveActiveTabTo(1);
b.moveActiveTabTo(1);
d.changeActiveTabTo(1);
b.moveActiveTabTo(3);
c.moveActiveTabTo(3);
d.moveActiveTabTo(3);
a.moveActiveTabTo(2);
d.moveActiveTabTo(2);
a.changeActiveTabTo(5);
b.changeActiveTabTo(5);
c.changeActiveTabTo(5);
d.changeActiveTabTo(5);
a.moveActiveTabTo(3);
b.changeActiveTabTo(3);
c.changeActiveTabTo(3);
d.moveActiveTabTo(3);
a.changeActiveTabTo(3);
b.moveActiveTabTo(3);
c.moveActiveTabTo(3);
d.moveActiveTabTo(3);
a.changeActiveTabTo(3);
b.changeActiveTabTo(3);
c.moveActiveTabTo(3);
d.changeActiveTabTo(3);
b.moveActiveTabTo(0);
d.moveActiveTabTo(0);
b.changeActiveTabTo(4);
a.changeActiveTabTo(3);
c.moveActiveTabTo(3);
d.moveActiveTabTo(3);
a.changeActiveTabTo(3);
c.moveActiveTabTo(3);
a.changeActiveTabTo(2);
d.changeActiveTabTo(2);
a.changeActiveTabTo(5);
c.changeActiveTabTo(5);
d.changeActiveTabTo(5);
a.moveActiveTabTo(4);
b.changeActiveTabTo(4);
a.changeActiveTabTo(3);
b.changeActiveTabTo(3);
a.moveActiveTabTo(3);
d.moveActiveTabTo(3);
a.moveActiveTabTo(0);
b.moveActiveTabTo(0);
b.moveActiveTabTo(1);
d.moveActiveTabTo(1);
a.moveActiveTabTo(0);
b.moveActiveTabTo(0);
c.moveActiveTabTo(0);
d.changeActiveTabTo(0);
a.moveActiveTabTo(3);
b.changeActiveTabTo(3);
a.changeActiveTabTo(4);
a.changeActiveTabTo(0);
b.moveActiveTabTo(0);
c.changeActiveTabTo(0);
d.changeActiveTabTo(0);
d.changeActiveTabTo(1);
b.changeActiveTabTo(4);
c.moveActiveTabTo(4);
d.changeActiveTabTo(4);
a.changeActiveTabTo(1);
b.moveActiveTabTo(1);
d.changeActiveTabTo(1);
a.changeActiveTabTo(1);
b.moveActiveTabTo(1);
b.changeActiveTabTo(2);
d.changeActiveTabTo(2);
a.moveActiveTabTo(1);
c.moveActiveTabTo(1);
d.moveActiveTabTo(1);
brows->addWindow(a);
brows->addWindow(b);
brows->addWindow(c);
brows->addWindow(d);
brows->closeEmptyWindows();
brows->closeWindow();
brows->newWindow();
brows->switchToWindow(3);
brows->newWindow();
brows->newWindow();
brows->switchToWindow(2);
brows->switchToWindow(9);
brows->print();
brows->closeWindow();
brows->mergeAllWindows();
brows->switchToWindow(6);
brows->getWindow(0);
brows->closeWindow();
brows->closeEmptyWindows();
brows->closeWindow();
brows->newWindow();
brows->newWindow();
brows->switchToWindow(4);
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->newWindow();
brows->newWindow();
brows->closeEmptyWindows();
brows->closeWindow();
brows->newWindow();
brows->switchToWindow(8);
brows->print();
brows->closeEmptyWindows();
brows->switchToWindow(8);
brows->switchToWindow(8);
brows->print();
brows->mergeAllWindows();
brows->getWindow(8);
brows->mergeAllWindows();
brows->getWindow(1);
brows->closeWindow();
brows->switchToWindow(6);
brows->mergeAllWindows();
brows->print();
brows->print();
brows->closeWindow();
brows->closeEmptyWindows();
brows->newWindow();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->getWindow(5);
brows->getWindow(4);
brows->mergeAllWindows();
brows->closeWindow();
brows->print();
brows->newWindow();
brows->getWindow(2);
brows->getWindow(9);
brows->closeWindow();
brows->switchToWindow(2);
brows->newWindow();
brows->closeEmptyWindows();
brows->closeWindow();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->closeWindow();
brows->getWindow(1);
brows->newWindow();
brows->switchToWindow(4);
brows->mergeAllWindows();
brows->closeWindow();
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->print();
brows->print();
brows->getWindow(4);
brows->getWindow(0);
brows->closeWindow();
brows->mergeAllWindows();
brows->closeWindow();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->getWindow(9);
brows->getWindow(1);
brows->switchToWindow(4);
brows->switchToWindow(1);
brows->print();
brows->closeWindow();
brows->newWindow();
brows->closeWindow();
brows->closeEmptyWindows();
brows->getWindow(0);
brows->closeEmptyWindows();
brows->newWindow();
brows->newWindow();
brows->closeEmptyWindows();
brows->print();
brows->newWindow();
brows->newWindow();
brows->print();
brows->newWindow();
brows->mergeAllWindows();
brows->getWindow(2);
brows->newWindow();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->newWindow();
brows->closeEmptyWindows();
brows->closeWindow();
brows->newWindow();
brows->closeWindow();
brows->newWindow();
delete brows;
}
	return 0;
}