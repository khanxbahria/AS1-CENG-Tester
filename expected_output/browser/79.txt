The active tab is tab 1
ddd
uu
n

a
kk
oo

tt
k
hhh





hhh
lll
m

The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 1
eee
z
yyy

aa
rr
bbb





kkk
x
ooo

s
yy
cc

l
kk
ww

i
qq
j

The active tab is tab 2
k
y
rr

v
o
qqq





xxx
yyy
uu

gg
iii
v

xx
xxx
k

xx
dd
hh

uu
j
qq

e
ww
p

The active tab is tab 1




w
w
dd

ll
qqq
k

eee
e
vv

g
www
ooo

cc
v
o

y
q
ss

p
ww
bbb

kkk
w
jj

The active tab is tab 0




The active tab is tab 0




The active tab is tab 1








The active tab is tab 0




The active tab is tab 0




The active tab is tab 1








Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
The browser is empty
The browser is empty
The browser is empty
The browser is empty
The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




